frontend/public/actions/ui.ts

```diff
@@ -31,6 +31,17 @@ export enum ActionType {
   SetActiveNamespace = 'setActiveNamespace',
   SetCreateProjectMessage = 'setCreateProjectMessage',
   SetCurrentLocation = 'setCurrentLocation',
+  SetServiceLevel = 'setServiceLevel',
+  SetServiceLevelDaysRemaining = 'SetServiceLevelDaysRemaining',
+  MonitoringDashboardsPatchVariable = 'monitoringDashboardsPatchVariable',
+  MonitoringDashboardsPatchAllVariables = 'monitoringDashboardsPatchAllVariables',
+  MonitoringDashboardsSetEndTime = 'monitoringDashboardsSetEndTime',
+  MonitoringDashboardsSetPollInterval = 'monitoringDashboardsSetPollInterval',
+  MonitoringDashboardsSetTimespan = 'monitoringDashboardsSetTimespan',
+  MonitoringDashboardsVariableOptionsLoaded = 'monitoringDashboardsVariableOptionsLoaded',
+  MonitoringSetRules = 'monitoringSetRules',
+  SetMonitoringData = 'setMonitoringData',
+  ToggleMonitoringGraphs = 'monitoringToggleGraphs',
```

Comment:
These might not need to be here.  I think they were accidentally introduced during a rebase.